### PR TITLE
Query multi tables

### PR DESCRIPTION
**Pull request by michaelamponsah**
### In this PR, the following requirements have been completed:

- [x] Created a table named owners with the following columns:
        id: integer (set it as autoincremented PRIMARY KEY)
        full_name: string
        age: integer  

- [x]  Created a table named species with the following columns:
        id: integer (set it as autoincremented PRIMARY KEY)
        name: string

- [x]  Modified animals table:
        Made sure that id is set as autoincremented PRIMARY KEY
        Removed column species
        Added column species_id which is a foreign key referencing species table
        Added column owner_id which is a foreign key referencing the ` owners table`

- [x] Executed all queries

> Below are screenshots demonstrating completed tasks. 

Capture

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,3 +12,23 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '7/06/2005', 7, true,  20.4);
```
Comment (Kamba56) [inline data.sql:12]: - [ ] Kindly edit all sections of your readme file and customize it to this project. Thsi will make your work more professional.
image
image
image
image

- [ ] Kindly fix the MIT license link in your readme file. The link should lead to the MIT license file in your repo.

File: queries.sql [hunk 2]
```diff
@@ -45,4 +45,26 @@ SELECT species, weight_kg AS max_weight FROM animals WHERE species = 'pokemon' A
...
+SELECT animals.name AS animal_name, owners.fullname AS owner FROM animals JOIN owners  ON animals.owner_id = owners.id WHERE owners.fullname = 'Melody Pond';
+
+-- Pokemon type animals
+SELECT animals.name AS animal_name, species.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+
+-- All owners and their animals
+SELECT owners.fullname AS owner, animals.name AS animal_name FROM animals RIGHT JOIN owners ON owners.id = animals.owner_id;
+
+--Animals per species
+SELECT species.name AS species, COUNT(animals.species_id) AS num FROM species  JOIN animals ON animals.species_id = species.id GROUP BY animals.species_id, species.name;
```
Comment (Kamba56) [inline queries.sql:60]: - [ ] All animals whose names end with `mon` are digimons. There are only 6 animals that end with `mon`. But from your screenshots, I see these returns 7, kindly look into that, please.

File: queries.sql [hunk 3]
```diff
@@ -45,4 +45,26 @@ SELECT species, weight_kg AS max_weight FROM animals WHERE species = 'pokemon' A
...
+
+-- All Digimon owned by Jennifer Orwell.
+SELECT owners.fullname AS owner, COUNT(animals.owner_id) AS digimon_count FROM owners JOIN animals ON owners.id = animals.owner_id WHERE owners.id = 2 AND animals.species_id = 2 GROUP BY owners.fullname, animals.owner_id;
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT animals.name AS no_escape_attempts, owners.fullname AS owner FROM animals JOIN owners ON animals.owner_id =  owners.id WHERE animals.owner_id = 5 AND animals.escape_attempts = 0 GROUP by owners.fullname, animals.name;
+
+-- Who owns the most animals
+SELECT  owners.fullname AS owner_name, MAX(animals.owner_id) AS owner_id FROM animals JOIN owners ON animals.owner_id = owners.id WHERE animals.owner_id = owners.id GROUP BY owners.fullname, animals.owner_id;
```
Comment (Kamba56) [inline queries.sql:69]: - [ ] Melody pond is the only owner that has 3 animals, so Melody should be returned as the person with the most animals, but that is not so in your screenshot, kindly fix the query please.

File: queries.sql [hunk 4]
```diff
@@ -45,4 +45,26 @@ SELECT species, weight_kg AS max_weight FROM animals WHERE species = 'pokemon' A
...
+
+-- All owners and their animals
+SELECT owners.fullname AS owner, animals.name AS animal_name FROM animals RIGHT JOIN owners ON owners.id = animals.owner_id;
+
+--Animals per species
+SELECT species.name AS species, COUNT(animals.species_id) AS num FROM species  JOIN animals ON animals.species_id = species.id GROUP BY animals.species_id, species.name;
+
+-- All Digimon owned by Jennifer Orwell.
+SELECT owners.fullname AS owner, COUNT(animals.owner_id) AS digimon_count FROM owners JOIN animals ON owners.id = animals.owner_id WHERE owners.id = 2 AND animals.species_id = 2 GROUP BY owners.fullname, animals.owner_id;
+
```
Comment (Kamba56) [inline queries.sql:64]: - [ ] Please list the animal's name that belongs to Jennifer Owrwell instead of counting the animals.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @michaelamponsah, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed
- [ ] Great job adding new tables and updating table entries

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @michaelamponsah     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**